### ISSUE TITLE
login page error after ssl install

### ISSUE THREAD

**Issue by ParallaxCoder** [litespeedtech/ols1clk#65]
i get 404 litespeed error for wordpress post urls and admin dashboard login page after running the below command to install SSL. (I replaced my domain with example.com). front page loads without a problem. 

`/bin/bash <( curl -sk https://raw.githubusercontent.com/litespeedtech/ls-cloud-image/master/Setup/vhsetup.sh ) -d www.example.com -le admin@example.com -f`

**Comment by ParallaxCoder** [comment]
i managed to fix this by deleting the newly created virtual host after running the above command. then adding SSL cert path manually to Listeners -> wordpressssl. also, note full chain didn't work either. I followed the below guide to edit that. please look into that I think vhsetup script conflicts with ols1clk wordpressplus install somehow.

https://www.litespeedtech.com/support/wiki/doku.php/litespeed_wiki:config:letsencrypt-ssl

**Comment by Code-Egg** [comment]
Hi @ParallaxCoder, I just tested it and it works for me, could you raise an issue to support@litespeedtech.com for further support?

**Comment by ParallaxCoder** [comment]
it's a kinda weird issue. I tried with centos7 and almalinux8 both had the same issue after running vhsetup.sh to add SSL to WordPress web. looks like it can't load .htaccess after that vhsetup.sh add another virtual host to the same domain as the WordPress domain. 

website works using https after running that but only the home page loading. all other pages including wp dashboard give me 404. I even tried to delete and manually create a new .htaccess file (chown correct group and user + permission) then restart lsws but nothing worked.

only workaround that worked is deleting the newly created virtual host and adding cert paths to Listeners -> wordpressssl. btw I used same certificate generated by vhsetup.sh nothing wrong with certificate. problem must be with new vhost generated by vhscript.sh

I think if vhscript doesn't add new vhost and directly modify Listeners -> wordpressssl to use generated certificate this issue can be mitigated or directly implement let's encrypt SSL install script to ols1clk? 

sorry for the wall of text. English is not my first language. it's hard to explain for me :(
